kart/point_cloud/import_.py

```diff
@@ -124,8 +137,39 @@ def point_cloud_import(ctx, ds_path, sources):
             "crs84_envelope": crs84_envelope,
         }
 
+        if schema is None:
+            crs_name = get_identifier_str(crs_set[0])
```

Comment:
minor thing but: these things are ListBasedSet instances - I understand the reason is so you can have a set that stores non-hashable things; ok fair enough

but then you're accessing them here quite unlike a set, relying on the underlying type as well as the ordering. I'd prefer `list(crs_set)[0]` or something I guess. Maybe you shouldn't be able to index a ListBasedSet at all since it's trying to be like a set

Reply:
I made a function .only() that does the same as [0] except that it asserts that it's a single-element-set, and I removed the __getitem__ operator.
It's a well-known fact that the first element of a single-element-set is reliably the only element, and that you can get it with next(iter(myset)) but that's too wordy. So I've compromised with .only() which is 7 characters compared to [0] which is three.